src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1251,14 +1251,59 @@ void TwitchMessageBuilder::appendTwitchBadges()
                 break;
             }
         }
+        else if (badge.startsWith("founder/"))
+        {
+            if (auto badgeEmote =
+                    this->twitchChannel->globalTwitchBadges().badge("founder",
+                                                                    "0"))
+            {
+                auto badgeInfo = this->tags.find("badge-info");
+                if (badgeInfo != this->tags.end() &&
+                    badgeInfo.value().toString().split(',')[0].startsWith(
```

Comment:
The `badgeInfo.value().toString().split(',')[0]` is used twice, so it can be moved.